From 21dcb3a98feb3ae6ba862cfbf25fba3361bc8a9b Mon Sep 17 00:00:00 2001
From: Martin Bark <martin@barkynet.com>
Date: Mon, 14 Dec 2015 21:46:17 +0000
Subject: [PATCH] unix: fix support for uClibc-ng

uClibc-ng is currently at v1.0.9.  The patch corrects the uClibc
version test so that HAVE_IFADDRS_H is defined for uClibc versions
after v0.9.32.

Signed-off-by: Martin Bark <martin@barkynet.com>
---
 src/unix/linux-core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/unix/linux-core.c b/src/unix/linux-core.c
index 3ff6fb1..4735bc4 100644
--- a/src/unix/linux-core.c
+++ b/src/unix/linux-core.c
@@ -39,7 +39,7 @@
 #define HAVE_IFADDRS_H 1
 
 #ifdef __UCLIBC__
-# if __UCLIBC_MAJOR__ < 0 || __UCLIBC_MINOR__ < 9 || __UCLIBC_SUBLEVEL__ < 32
+# if __UCLIBC_MAJOR__ < 0 && __UCLIBC_MINOR__ < 9 && __UCLIBC_SUBLEVEL__ < 32
 #  undef HAVE_IFADDRS_H
 # endif
 #endif
-- 
2.5.0

